### PR TITLE
[4.1] content params

### PR DESCRIPTION
**Pull request by brianteeman**
In the installation only some of the default params for com_content are setup in the database. It's not really a problem as there are default values BUT it means that where you have a setting in a menu item that is "Use Global" it doesn't display the value.

#### Before
image

#### After
image

This is only for the installation.

### CODE REVIEW

File: installation/sql/postgresql/base.sql [hunk 1]
```diff
@@ -172,7 +172,7 @@ INSERT INTO "#__extensions" ("package_id", "name", "type", "element", "folder",
...
...s_images_frontend":"0","show_urls_images_backend":"1","targeta":0,"targetb":0,"targetc":0,"float_intro":"left","float_fulltext":"left","category_layout":"_:blog","show_category_title":"0","show_description":"0","show_description_image":"0","maxLevel":"1","show_empty_categories":"0","show_no_articles":"1","show_category_heading_title_text":"1","show_subcat_desc":"1","show_cat_num_articles":"0","show_cat_tags":"1","show_base_description":"1","maxLevelcat":"-1","show_empty_categories_cat":"0","show_subcat_desc_cat":"1","show_cat_num_articles_cat":"1","num_leading_articles":1,"blog_class_leading":"","num_intro_articles":4,"blog_class":"","num_columns":1,"multi_column_order":"0","num_links":4,"show_subcategory_content":"0","link_intro_image":"0","show_pagination_limit":"1","filter_field":"hide","show_headings":"1","list_show_date":"0","date_format":"","list_show_hits":"1","list_show_author":"1","display_num":"10","orderby_pri":"order","orderby_sec":"rdate","order_date":"published","show_pagination":"2","show_pagination_results":"1","show_featured":"show","show_feed_link":"1","feed_summary":"0","feed_show_readmore":"0","sef_ids":1,"custom_fields_enable":"1","workflow_enabled":"0"}', ''),
```
Comment (richard67) [inline installation/sql/postgresql/base.sql]:
```suggestion
(0, 'com_content', 'component', 'com_content', '', 1, 1, 0, 1, 1, '', '{"article_layout":"_:default","show_title":"1","link_titles":"1","show_intro":"1","info_block_position":"0","info_block_show_title":"1","show_category":"1","link_category":"1","show_parent_category":"0","link_parent_category":"0","show_associations":"0","flags":"1","show_author":"1","link_author":"0","show_create_date":"0","show_modify_date":"0","show_publish_date":"1","show_item_navigation":"1","show_readmore":"1","show_readmore_title":"1","readmore_limit":100,"show_tags":"1","record_hits":"1","show_hits":"1","show_noauth":"0","urls_position":0,"captcha":"","show_publishing_options":"1","show_article_options":"1","show_configure_edit_options":"1","show_permissions":"1","show_associations_edit":"1","save_history":"1","history_limit":10,"show_urls_images_frontend":"0","show_urls_images_backend":"1","targeta":0,"targetb":0,"targetc":0,"float_intro":"left","float_fulltext":"left","category_layout":"_:blog","show_category_title":"0","show_description":"0","show_description_image":"0","maxLevel":"1","show_empty_categories":"0","show_no_articles":"1","show_category_heading_title_text":"1","show_subcat_desc":"1","show_cat_num_articles":"0","show_cat_tags":"1","show_base_description":"1","maxLevelcat":"-1","show_empty_categories_cat":"0","show_subcat_desc_cat":"1","show_cat_num_articles_cat":"1","num_leading_articles":1,"blog_class_leading":"","num_intro_articles":4,"blog_class":"","num_columns":1,"multi_column_order":"0","num_links":4,"show_subcategory_content":"0","link_intro_image":"0","show_pagination_limit":"1","filter_field":"hide","show_headings":"1","list_show_date":"0","date_format":"","list_show_hits":"1","list_show_author":"1","display_num":"10","orderby_pri":"order","orderby_sec":"rdate","order_date":"published","show_pagination":"2","show_pagination_results":"1","show_featured":"show","show_feed_link":"1","feed_summary":"0","feed_show_readmore":"0","sef_ids":1,"custom_fields_enable":"1","workflow_enabled":"0"}', '', 0, 0),
```
In the PostgreSQL file we have here the "ordering" and "state" columns in the statement, in opposite to the MySQL file. Not having them in the values causes an SQL error and so installation fails in system tests for PostgreSQL.

Comment (richard67) [reply installation/sql/postgresql/base.sql]: InstallCest installJoomla postgres fail

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 04465e31ccc6990b727e7dc84307b6e313ad1f3c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37288">issues.joomla.org/tracker/joomla-cms/37288</a>.</sub>

**Comment by richard67** [comment]
Hint for other testers: Just make a new installation with the patch of this PR applied. Then verify that the params of the com_content extension in database are the same as before when you have opened and then saved the com_content parameters in backend.

**Comment by RickR2H** [comment]
I have tested this item :white_check_mark: successfully on 04465e31ccc6990b727e7dc84307b6e313ad1f3c<br /><br />Installed the patch before install and everything seems to work. Default setting value is shown.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37288">issues.joomla.org/tracker/joomla-cms/37288</a>.</sub>

**Comment by RickR2H** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37288">issues.joomla.org/tracker/joomla-cms/37288</a>.</sub>

**Comment by brianteeman** [comment]
Note that there are many more components with this problem. But not very mmotivated to fix them when the pull request is not merged

**Comment by Quy** [comment]
Thank you!1

**Comment by Quy** [comment]
Thank you!

**Pull request merged**